#! /bin/sh /usr/share/dpatch/dpatch-run
## 11_check_http_arguments.dpatch by Jan Wagner <waja@cyconet.org>
##
## From c91ff7539af65f95c2bcef69e5643bd99773c80f Mon Sep 17 00:00:00 2001
## From: Holger Weiss <holger@zedat.fu-berlin.de>
## Date: Thu, 5 Dec 2013 20:13:31 +0100
## Subject: [PATCH] check_http: Don't let "-N" expect an argument
##
## Fix the problem that check_http's "-N" option expects an argument
## although it shouldn't.  The corresponding long option ("--no-body")
## wasn't affected.
##
## This error was introduced in ee3c4014d511e105489b081390636cacd4c53e3f.
## Thanks to Kerry Hughes for reporting it on the devel@ list.

@DPATCH@

diff --git a/plugins/check_http.c b/plugins/check_http.c
index 0ffb0c1..1595a59 100644
--- a/plugins/check_http.c
+++ b/plugins/check_http.c
@@ -257,7 +257,7 @@ enum {
   }
 
   while (1) {
-    c = getopt_long (argc, argv, "Vvh46t:c:w:A:k:H:P:j:T:I:a:b:d:e:p:s:R:r:u:f:C:J:K:nlLS::m:M:N:E", longopts, &option);
+    c = getopt_long (argc, argv, "Vvh46t:c:w:A:k:H:P:j:T:I:a:b:d:e:p:s:R:r:u:f:C:J:K:nlLS::m:M:NE", longopts, &option);
     if (c == -1 || c == EOF)
       break;
 
-- 
1.8.5.1

